README.md

```diff
@@ -0,0 +1,61 @@
+# About workshop
+
+In this workshop we will explore what Firebase Functions are and setup project to be able to write those functions in Kotlin. To get there we will touch some other interesting things like Realtime Database and TypeScript. 
+
+
+### Requirements:
+
+ * [Firebase account](https://console.firebase.google.com/)
+ * Code editor for JavaScript/TypeScript (my choice is [VS Code](https://code.visualstudio.com/))
+ * [IntelliJ Idea community edition](https://www.jetbrains.com/idea/download/)
+
+
+### What are we building:
+
+> **Ultimate Currency converter**
+> 
+> Let's make a backend for the ultimate currency convertion app - user enters price in euro and backend converts it to other currencies.
+
+Backend will work as follows: 
+ * Client app pushes new "conversion" with euro value into table
```

Comment:
The Client app/The app/The client/The app (a.k.a. client)
a new
a euro value
the table (what table? - the word_missing table)